Documentation/Books/Manual/ReleaseNotes/KnownIssues33.md

```diff
@@ -6,11 +6,12 @@ should take note of. It is not a list of all open issues.
 
 ArangoDB Customers can access the
 [_Technical & Security Alerts_](https://arangodb.atlassian.net/wiki/spaces/DEVSUP/pages/223903745)
-page after login in the Support Portal.
+page after login into the Support Portal.
 
 | # | Issue      |
 |---|------------|
-| 4 | **Date Added:** 2018-11-30 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Wrong suggestion printed in the log on how to optimize an OS setting, if followed, could cause ArangoDB to run into problems as the number of memory mappings will keep growing <br> **Affect Versions:** 3.3.0 to 3.3.19 <br> **Fixed in Versions:** 3.3.20 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/228622337)  |
-| 3 | **Date Added:** 2018-11-16 <br> **Component:** Security <br> **Deployment Mode:** All <br> **Description:** Unauthorized access to ArangoDB when using LDAP authentication <br> **Affect Versions:** 3.3.0 to 3.3.18 <br> **Fixed in Versions:** 3.3.19 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/223903752)  |
-| 2 | **Date Added:** 2018-11-03 <br> **Component:** Backup/Restore <br> **Deployment Mode:** All <br> **Description:** Users not included in the backup if _--server.authentication = true_ <br> **Affect Versions:** 3.3.0 to 3.3.13 <br> **Fixed in Versions:** 3.3.14 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/226557953) |
-| 1 | **Date Added:** 2018-04-09 <br> **Component:** Storage <br> **Deployment Mode:** Single Instance <br> **Description:** Data corruption could happen under Linux <br> **Affect Versions:** 3.3.0 <br> **Fixed in Versions:** 3.3.1 <br> **Reference:** [Link for ArangoDB Customers](https://arangodb.atlassian.net/servicedesk/customer/kb/view/164069377)  |
+| 5 | **Date Added:** 2018-12-04 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Parallel creation of collections using multiple client connections with the same database user may spuriously fail with "Could not update user due to conflict" warnings when setting user permissions on the new collections. A follow-up effect of this may be that access to the just-created collection is denied. <br> **Affected Versions:** 3.3 series ** <br> **Fixed in Versions:** 3.4.1 <br> **Reference:** [arangodb/arangodb#5342](https://github.com/arangodb/arangodb/issues/5342)  |
```

Comment:
Extra `**` after affected versions.

Should be `3.3.x, 3.4.0` I guess? Or is there a fix in 3.3.20 or something? Then it should be `3.3.0 to 3.3.19, 3.4.0` (following Santo's convention from the other issues).

```suggestion
| 5 | **Date Added:** 2018-12-04 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Parallel creation of collections using multiple client connections with the same database user may spuriously fail with "Could not update user due to conflict" warnings when setting user permissions on the new collections. A follow-up effect of this may be that access to the just-created collection is denied. <br> **Affected Versions:** 3.3.x, 3.4.0 <br> **Fixed in Versions:** 3.4.1 <br> **Reference:** [arangodb/arangodb#5342](https://github.com/arangodb/arangodb/issues/5342)  |
```